src/App/App.tsx

```diff
@@ -119,332 +119,317 @@ const APP_LOGIN_URL = BACKEND_ACTIVE ? BACKEND_LOGIN_URL : REACT_LOGIN_URL;
+              />
+              {/* single organization view */}
+              <ConnectedPrivateRoute
+                redirectPath={APP_CALLBACK_URL}
+                disableLoginProtection={DISABLE_LOGIN_PROTECTION}
+                exact={true}
+                path={`${SINGLE_ORGANIZATION_URL}/:id`}
+                component={ConnectedSingleOrgView}
+              />
+              {/* Practitioner listing page */}
+              <ConnectedPrivateRoute
+                redirectPath={APP_CALLBACK_URL}
+                disableLoginProtection={DISABLE_LOGIN_PROTECTION}
+                exact={true}
+                path={PRACTITIONERS_LIST_URL}
+                component={ConnectedPractitionersListView}
+              />
+              {/** practitioner create view */}
+              <ConnectedPrivateRoute
+                redirectPath={APP_CALLBACK_URL}
+                disableLoginProtection={DISABLE_LOGIN_PROTECTION}
+                exact={true}
+                path={CREATE_PRACTITIONER_URL}
+                component={ConnectedCreateEditPractitionerView}
+              />
+              {/** Practitioner edit view */}
+              <ConnectedPrivateRoute
+                redirectPath={APP_CALLBACK_URL}
+                disableLoginProtection={DISABLE_LOGIN_PROTECTION}
+                exact={true}
+                path={`${EDIT_PRACTITIONER_URL}/:id`}
+                component={ConnectedCreateEditPractitionerView}
+              />
+              {/** Assign practitioners to organization view */}
+              />
+              <ConnectedPrivateRoute
+                redirectPath={APP_CALLBACK_URL}
+                disableLoginProtection={DISABLE_LOGIN_PROTECTION}
+                exact={true}
+                path={`${ASSIGN_PRACTITIONERS_URL}/:id`}
+                component={ConnectedAssignPractitioner}
+              />
+              {/* tslint:disable jsx-no-lambda */}
+              <Route
+                exact={true}
+                path={APP_LOGIN_URL}
+                render={routeProps => (
+                  <OauthLogin
+                    providers={providers}
+                    authorizationGrantType={AuthGrantType}
+                    {...routeProps}
+                  />
+                )}
+              />
+              <Route
+                exact={true}
+                path={APP_CALLBACK_PATH}
+                render={routeProps => {
+                  if (BACKEND_ACTIVE) {
+                    return <CustomConnectedAPICallBack {...routeProps} />;
+                  }
+                  return (
+                    <ConnectedOauthCallback
+                      SuccessfulLoginComponent={() => {
+                        return <Redirect to={HOME_URL} />;
+                      }}
+                      LoadingComponent={Loading}
                       providers={providers}
-                      authorizationGrantType={AuthGrantType}
+                      oAuthUserInfoGetter={oAuthUserInfoGetter}
```

Comment:
Why is this changing?

Reply:
Git is taking it as a change but have looked at it and nothing changed. Initially that is were the `login component` was before the lines reduced. Now the `home component` has moved to that position and i think that is why it was registered as a change.